src/main/java/uk/gov/hmcts/divorce/caseworker/service/task/SendAosResponseLetterPackToApplicant.java

```diff
@@ -23,8 +23,14 @@
         final CaseData caseData = caseDetails.getData();
         final AcknowledgementOfService acknowledgementOfService = caseData.getAcknowledgementOfService();
 
-        if (caseData.getApplicant1().isOffline() && acknowledgementOfService.isDisputed()) {
-            log.info("Sending aos response letter pack to bulk print as applicant1 is offline. Case id: {}", caseId);
+        if (caseData.getApplicant1().isOffline()) {
```

Comment:
Are we sure the AoS response only goes to offline applicants?

Reply:
Yes Linus, As per the ACs in [NFDIV-1717](https://tools.hmcts.net/jira/browse/NFDIV-1717) and [NFDIV-1718](https://tools.hmcts.net/jira/browse/NFDIV-1718) the AoS response letter is only sent if applicant is flagged as offline.

The email notifications for online applicants are sent separately, as part of task **SendApplicationIssueNotifications**